.github/workflows/terraform-lint.yml

```
@@ -0,0 +1,47 @@
+      - name: Set up TFLint
+        uses: terraform-linters/setup-tflint@v4
+        with:
+          tflint_version: "v0.53.0"
+
+      - name: TFLint init
+        run: tflint --init --config=infra/.tflint.hcl
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: TFLint
+        run: tflint --chdir=infra/environments/dev --config=$(pwd)/infra/.tflint.hcl
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:48: Quote this to prevent word splitting

(shellcheck)